lib/spring/application.rb

```diff
@@ -247,6 +251,18 @@ def loaded_application_features
       $LOADED_FEATURES.select { |f| f.start_with?(root) }
     end
 
+    def loaded_application_dependencies
+      root = Spring.application_root_path.to_s
+      loaded = ActiveSupport::Dependencies.loaded
+
+      # The loaded_paths do not include file suffixes. Find all matches.
+      dlexts = RbConfig::CONFIG.values_at('DLEXT', 'DLEXT2')
+      ext_glob = "{rb,#{dlexts.reject { |s| s.to_s.empty? }.join(',')}}"
```

Comment:
if there are no dlext then this would be `{.rb,}` is that right ?

maybe `extensions = ['.rb'] + RbConfig::CONFIG.values_at('DLEXT', 'DLEXT2').reject(&:empty)`